mesa/visualization/solara_viz.py

```diff
@@ -109,20 +107,27 @@ def step():
 
     solara.use_effect(connect_to_model, [model.value])
 
+    # set up reactive model_parameters shared by ModelCreator and ModelController
+    solara.use_effect(
```

Comment:
On your question about the "need" to move this here: Can't you just initialize model_parameters as an empty dict here? 
`model_parameters = solara.reactive({})` And then the rest of the logic can stay in ModelCreator